apps/minifront/src/components/ibc/deposit-manual/hooks.ts

```diff
@@ -32,50 +19,23 @@ export const useChainConnector = () => {
   return useChain(selectedOrDefaultChain);
 };
 
-const useCosmosQueryHooks = () => {
+const useCosmosChainBalance = () => {
   const { address, getRpcEndpoint, chain } = useChainConnector();
 
-  const rpcEndpointQuery = useRpcEndpoint({
-    getter: getRpcEndpoint,
-    options: {
-      enabled: !!address,
-      staleTime: Infinity,
-      queryKey: ['rpc endpoint', address, chain.chain_name],
-      // Needed for osmo-query's internal caching
-      queryKeyHashFn: queryKey => {
-        return JSON.stringify([...queryKey, chain.chain_name]);
-      },
+  return useQuery({
+    queryKey: ['rpc endpoint', address, chain.chain_name],
+    queryFn: async () => {
+      if (!address) {
+        throw new Error('missing address');
+      }
+      const endpoint = await getRpcEndpoint();
+      const client = await StargateClient.connect(endpoint);
+      return client.getAllBalances(address);
     },
-  }) as UseQueryResult<string>;
-
-  const rpcClientQuery = useRpcClient({
-    rpcEndpoint: rpcEndpointQuery.data ?? '',
-    options: {
-      enabled: !!address && !!rpcEndpointQuery.data,
-      staleTime: Infinity,
-      queryKey: ['rpc client', address, rpcEndpointQuery.data, chain.chain_name],
-      // Needed for osmo-query's internal caching
-      queryKeyHashFn: queryKey => {
-        return JSON.stringify([...queryKey, chain.chain_name]);
-      },
-    },
-  }) as UseQueryResult<ProtobufRpcClient>;
-
-  const { cosmos: cosmosQuery, osmosis: osmosisQuery } = createRpcQueryHooks({
-    rpc: rpcClientQuery.data,
+    enabled: !!address,
```

Comment:
Far simpler than what the former osmo-query helpers were providing

Reply:
using standard `useQuery` over osmo-query specific hooks also has the benefit of reducing reliance on third-party libs, which minimizes another potential source of breakage.